RM_SUI/RM_SUI/ContentView.swift

```diff
@@ -8,16 +8,218 @@
+                                .font(.title3)
+                        }.pickerStyle(.navigationLink)
+                    }
+                    
+                    HStack {
+                        Image(systemName: Constant.antennaImageName)
+                            .frame(width: CGFloat(Constant.badgesWidthHeight),
+                                   height: CGFloat(Constant.badgesWidthHeight))
+                            .foregroundColor(.white)
+                            .background(.blue)
+                            .cornerRadius(CGFloat(Constant.cornerRadius))
+                        
+                        Picker(selection: $cellularSection) {
+                            ForEach(0..<onOfParam.count, id: \.self) {
+                                Text(self.onOfParam[$0])
+                            }
+                        } label: {
+                            Text(Constant.cellularPickerTextName)
+                                .font(.title3)
+                        }.pickerStyle(.navigationLink)
+                    }
+                    
+                    HStack {
+                        Image(systemName: Constant.hotspotImageName)
+                            .frame(width: CGFloat(Constant.badgesWidthHeight),
+                                   height: CGFloat(Constant.badgesWidthHeight))
+                            .foregroundColor(.white)
+                            .background(.green)
+                            .cornerRadius(CGFloat(Constant.cornerRadius))
+                        
+                        Picker(selection: $modemSection) {
+                            ForEach(0..<onOfParam.count, id: \.self) {
+                                Text(self.onOfParam[$0])
+                            }
+                        } label: {
+                            Text(Constant.hotspotModePickerText)
+                                .font(.title3)
+                        }.pickerStyle(.navigationLink)
+                    }
+                    
+                    HStack {
+                        Image(systemName:
+                                Constant.airTagImageName)
+                        .foregroundColor(.white)
+                        .frame(width: CGFloat(Constant.badgesWidthHeight),
+                               height: CGFloat(Constant.badgesWidthHeight))
+                        .background(.blue)
+                        .cornerRadius(CGFloat(Constant.cornerRadius))
+                        Toggle(isOn: $VPNIsOn) {
+                            Text(Constant.vpnModeName)
+                                .font(.title3)
+                        }
+                    }
+                }
+            }.navigationTitle(Text(Constant.NavigationTitle))
         }
-        .padding()
     }
 }
 
 struct ContentView_Previews: PreviewProvider {
     static var previews: some View {
         ContentView()
+            .previewDevice("iPhone 14 Pro")
```

Comment:
```suggestion
            .previewDevice("iPhone 14 Pro")
```
вот тут не уверен стоит ли в константы выносить, когда значение только одно... Но если думать о расширении приложения, то надо бы конечно